src/server/api/lib/nexmo.js

```diff
@@ -177,8 +177,9 @@ async function handleIncomingMessage(message) {
 
   const pendingMessagePart = new PendingMessagePart({
     service: 'nexmo',
-    parent_id: parentId,
-    service_message: message,
+    service_id: message['concat-ref'] || message.messageId,
+    parent_id: parentId, //do we need this anymore, now we have service_id?
```

Comment:
i don't think we need it anymore ---> parentId ( parentId = message['concat-ref'] ) was previously created the same way so I assume it's serving the same purpose as service_id